### ISSUE TITLE
Storing signals with relayer

### ISSUE THREAD

**Issue by JohnCoperman** [MehVahdJukaar/Supplementaries#807]
MC v 1.19.4 fabric 0.14.23 Mod v 2.4.15-fabric (moonlight 2.4.16-fabric)

I personally don't consider this a bug, but a feature. However, you are the developer, you decide what to be and what not.

We know that the relayer can't see the signal on a normal (full) block. But if the block is placed and gets a signal at the same time,
- the relayer will see that signal (strong or weak). And moreover - it saves it! You can turn off the signal on the block.

That state is inert to other signals, leaving the world or game. Also, is inert to other block updates around. But if you break the block or move it, or move a relayer, the stored signal is gone.
Is figure out, that relayer must see the block update to cach the signal. Normal blocks havent updates on redstone, unless placing/moving or break.

If the stored signal disappeared in 0-tic time, this might be more useful..... I have not been able to test this in game. I've tried it, but no success. But I got results from which I can tell that the relayer shuts down around 1-tick time. Always.

screenshots:
![supplem_relayer-cell_1](https://github.com/MehVahdJukaar/Supplementaries/assets/111206434/cd1cb0a6-e5f4-49ad-9312-ff6be1f92219)
![supplem_relayer-cell_2](https://github.com/MehVahdJukaar/Supplementaries/assets/111206434/2720dba2-25c5-45b0-b87f-e06efe21f36d)
![supplem_relayer-cell_3](https://github.com/MehVahdJukaar/Supplementaries/assets/111206434/90925397-19e6-461d-8f70-a142cea0766d)
![supplem_relayer-cell_3-0](https://github.com/MehVahdJukaar/Supplementaries/assets/111206434/990ffd41-ffcb-44aa-9df7-734f819057a3)
![supplem_relayer-cell_3](https://github.com/MehVahdJukaar/Supplementaries/assets/111206434/b4d7f2dd-896d-403f-ba43-c07843f9296b)
![supplem_relayer-cell_5](https://github.com/MehVahdJukaar/Supplementaries/assets/111206434/f50493b9-9629-4fcd-9eb3-ffcad53248de)
![supplem_relayer-cell_6](https://github.com/MehVahdJukaar/Supplementaries/assets/111206434/16baf277-426a-4786-a638-8370ae9eba29)

It turns out that in order to catch a signal, relayer must see not only the signal, but also the block update. Normal blocks won't update from a signal unless they are placed, moved, or broken. And that's not a bug, is it?!

**Comment by MehVahdJukaar** [comment]
Yeah it's a bug. It's meant to copy whatever singla that target block has. Works on place but doesn't get the proper block update it seems. Idk if that's a useful thing to have, if not I'll try to fix

**Comment by JohnCoperman** [comment]
This can be useful. We can dynamically store the signal, also with its level. ! Composters will lose here, they can be filled automatically, but it's more complicated and longer.
I think it would be good for code locks on redstone. Or other mechanisms that depend on the order of the input signals.

**Comment by MehVahdJukaar** [comment]
hm ill leave this open. guess ill keep it as a sort of feature for now

**Comment by meeprdib** [comment]
Mod versions on 1.19.2 and below are not supported

**Comment by MehVahdJukaar** [comment]
Leaving this open as this is actually something to look into further
